src/lib/SX127xDriver/SX127xRegs.h

```diff
@@ -24,13 +24,14 @@ typedef enum
 typedef enum
 {
     // Only 125 to 500 KHZ supported
-    SX127x_BW_7_80_KHZ = 0b00000000,
-    SX127x_BW_10_40_KHZ = 0b00000000,
-    SX127x_BW_15_60_KHZ = 0b00000000,
-    SX127x_BW_20_80_KHZ = 0b00000000,
-    SX127x_BW_31_25_KHZ = 0b00000000,
-    SX127x_BW_41_70_KHZ = 0b00000000,
-    SX127x_BW_62_50_KHZ = 0b00000000,
+    // 7 to 62KHZ are unique so switch cases don't complain about duplicate values
```

Comment:
How about just removing the unused values?

Reply:
So there's 2 requirements for the build to succeed. The cases need to exist since they're used in the switch and they need to be unique or the compiler complains. I'm open to alternative suggestions and here's 2 example errors I was running into.

1. `lib/SX127xDriver/SX127x.cpp:621:8: error: 'SX127x_BW_41_70_KHZ' was not declared in this scope; did you mean 'SX127x_BW_500_00_KHZ'?`
2. `lib/SX127xDriver/SX127x.cpp:625:3: error: duplicate case value`

Reply:
Remove the cases from the cpp code as well.
If we need to support these at a later stage we can re-add them then.

Reply:
Okay removed, I didn't know they weren't being used.